org.freecadweb.FreeCAD.yaml

```diff
@@ -661,26 +661,28 @@ modules:
     post-install:
       - install -Dm755 ../openscad.sh ${FLATPAK_DEST}/bin/openscad
       - ln -s /app/freecad/bin/FreeCADCmd /app/bin/FreeCADCmd
-      - install -D ../FreeCAD.sh /app/bin/FreeCAD
+      - ln -s /app/freecad/bin/FreeCAD /app/bin/FreeCAD
       # The mimetype icon to be exported.
       - install -Dm644 ../src/Gui/Icons/freecad-doc.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/mimetypes/${FLATPAK_ID}.application-x-extension-fcstd.svg
     sources:
-      - type: script
-        dest-filename: FreeCAD.sh
-        commands:
-          - exec /app/freecad/bin/FreeCAD -u ${XDG_CONFIG_HOME}/FreeCAD/user.conf -s ${XDG_CONFIG_HOME}/FreeCAD/system.cfg "$@"
       - type: git
         url: https://github.com/FreeCAD/FreeCAD.git
         tag: '0.20beta1'
-        # commit: 59e1836c9bd613ef9264740da9755b33ac5823a6
       - type: file
         path: openscad.sh
       # we need this patch so that FreeCAD can find QAssistant
       - type: patch
         path: patches/freecad-assistant-path.patch
       - type: shell
         commands:
-          - sed -i 's|@PACKAGE_VERSION@|0.20beta1|' src/XDGData/org.freecadweb.FreeCAD.appdata.xml.in
+          # revcount needs to be updated any time the source commit or tag is changed
+          # to obtain the revcount for a given gitRef (commit hash, branch or tag) run
+          # echo $(($(curl -s 'https://api.github.com/repos/FreeCAD/FreeCAD/compare/120ca87015...'"$gitRef"'' | grep "ahead_by" | sed -s 's/ //g;s/"ahead_by"://;s/,//')+1))
+          - |
+            revcount=28936
+            sed -i 's|${PACKAGE_WCREF}|'"$revcount"' (Git)|' src/Build/Version.h.cmake
+            sed -i 's|@PACKAGE_VERSION@|@PACKAGE_VERSION@.'"$revcount"'|' src/XDGData/org.freecadweb.FreeCAD.appdata.xml.in
+          - sed -i 's|${PACKAGE_WCURL}|git://github.com/FreeCAD/FreeCAD.git|' src/Build/Version.h.cmake
```

Comment:
not sure how relevant that is, but Github deprecated `git://` which make this repository URL invalid.

Reply:
In the about dialog this let's freecad make a link to the git commit from which it was built, when normally compiling freecad it produces that output and it works anyways, this line isn't really necessary when using a git source because it picks it from the git remote, it's only useful if you build from source tarball